lib/tasks/data_hygiene.rake

```diff
@@ -23,4 +23,13 @@ namespace :data_hygiene do
       call_change_note_remover(args[:content_id], args[:locale], args[:query], dry_run: false)
     end
   end
+
+  desc "Check the status of a document whether it's in Content Store or Router."
+  task :document_status_check, %i[content_id locale] => :environment do |_, args|
+    document = Document.find_by!(content_id: args[:content_id], locale: args[:locale])
+    status = DataHygiene::DocumentStatusChecker.new(document)
+
+    puts "Content Store? #{status.content_store?}"
+    puts "Router? #{status.router?}"
```

Comment:
Is it possible to have more descriptive output here?  I'm not sure to someone approaching it for the first time (e.g. me) that e.g. "Content Store? true" means that it's in the content store, rather than actually *being* the content store, or published by the content store, or whatever.